metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorModule.java

```diff
@@ -55,7 +57,9 @@ public HiveConnectorModule(final String catalogName, final Map<String, String> c
         this.catalogName = catalogName;
         this.infoConverter = infoConverter;
         this.hiveMetastoreClientFactory =
-                new HiveMetastoreClientFactory(null);
+                new HiveMetastoreClientFactory(null,
+                        (int) HiveConnectorUtil.toTime(configuration.getOrDefault(hivemetastoreTimeout, "20s"),
```

Comment:
Can we consider the configuration as time in seconds. This way you dont have to parse the string for the unit.